src/examples/ex12.c

```diff
@@ -71,7 +71,7 @@ int main (int argc, char *argv[])
    HYPRE_Init();
 
    /* Print GPU info */
-   HYPRE_PrintDeviceInfo();
+   /* HYPRE_PrintDeviceInfo(); */
```

Comment:
Why comment this out vs removing it altogether?

Reply:
Wanted to remind users they can print out info of GPU that is used by that. But turning it on pollutes the output, which fails regression tests.

Reply:
Works for me.  Thanks!